package.json

```
@@ -3,6 +3,9 @@
   "version": "1.1.3",
   "description": "Convert between schema specifications",
   "main": "bindings/node/index.js",
+  "scripts": {
+    "start": "./node_modules/webpack-dev-server/bin/webpack-dev-server.js --config webpack.config.dev.js --open --port 8080"
```

```suggestion
    "start": "make serve"
```

To avoid duplication?